tests/test_google_ads_bookmarks.py

```diff
@@ -137,71 +166,48 @@ def test_run(self):
 
                 if expected_replication_method == self.INCREMENTAL:
 
-                    # included to catch a contradiction in our base expectations
-                    if not self.is_report(stream):
-                        raise AssertionError(
-                            f"Only Reports streams should be expected to support {expected_replication_method} replication."
-                        )
-
+                    # gather expectations
                     expected_replication_key = list(self.expected_replication_keys()[stream])[0]  # assumes 1 value
                     manipulated_bookmark = manipulated_state['bookmarks'][stream]
-                    today_minus_conversion_window = dt.utcnow().replace(hour=0, minute=0, second=0, microsecond=0) - conversion_window
-                    today = dt.utcnow().replace(hour=0, minute=0, second=0, microsecond=0)
                     manipulated_state_formatted = dt.strptime(manipulated_bookmark.get(expected_replication_key), self.REPLICATION_KEY_FORMAT)
-                    if manipulated_state_formatted < today_minus_conversion_window:
-                        reference_time = manipulated_state_formatted
-                    else:
-                        reference_time = today_minus_conversion_window
 
                     # Verify bookmarks saved match formatting standards for sync 1
                     self.assertIsNotNone(stream_bookmark_1)
                     bookmark_value_1 = stream_bookmark_1.get(expected_replication_key)
                     self.assertIsNotNone(bookmark_value_1)
                     self.assertIsInstance(bookmark_value_1, str)
-                    try:
-                        parsed_bookmark_value_1 = dt.strptime(bookmark_value_1, self.REPLICATION_KEY_FORMAT)
-                    except ValueError as err:
-                        raise AssertionError(
-                            f"Bookmarked value does not conform to expected format: {self.REPLICATION_KEY_FORMAT}"
-                        ) from err
+                    self.assertIsDateFormat(bookmark_value_1, self.REPLICATION_KEY_FORMAT)
 
                     # Verify bookmarks saved match formatting standards for sync 2
                     self.assertIsNotNone(stream_bookmark_2)
                     bookmark_value_2 = stream_bookmark_2.get(expected_replication_key)
                     self.assertIsNotNone(bookmark_value_2)
                     self.assertIsInstance(bookmark_value_2, str)
+                    self.assertIsDateFormat(bookmark_value_2, self.REPLICATION_KEY_FORMAT)
 
-                    try:
-                        parsed_bookmark_value_2 = dt.strptime(bookmark_value_2, self.REPLICATION_KEY_FORMAT)
-                    except ValueError as err:
-
-                        try:
-                            parsed_bookmark_value_2 = dt.strptime(bookmark_value_2, "%Y-%m-%dT%H:%M:%S.%fZ")
-                        except ValueError as err:
```

Comment:
Check, I assume, per Dylan discussion above.

Reply:
Yes. This was necessary before, because the of the time info attached when the conversion window was used. Now that we aren't going to use the time info and just truncate to midnight, that's not an issue.